README.md

```diff
@@ -1,67 +1,139 @@
 # cli-transparent-tunnel 
 
-* CLI wrapper that:
-  * create a ssl tunnel -> run the command through the tunnel -> close the tunnel.
-  * basically it can act as an inline replacer fot the CLI 
-    * the wrapper check if there is a tunnel configured for the required endpoint, if it has:
-      * it adds the ssh tunnel flags to the CLI and invoke the command with all args.
-      * if not, it just invokes the CLI 
+## Introduction
 
-* THIS PROJECT is WIP, I believe it can solve accessing k8s clusters through ssh tunnel in certain scenarios such as CI / CD etc .
-* Let me know what you think.
+SSH tunneling is a great method of transporting arbitrary networking data over an encrypted SSH connection.
+It can be used to add 
```

Comment:
Consider rephrase to: "Using SSH tunneling, you can transport arbitrary data over an encrypted SSH connection.  Main use-cases are:"